aviatrix/resource_aviatrix_firewall_instance.go

```diff
@@ -388,6 +396,10 @@ func resourceAviatrixFirewallInstanceCreate(d *schema.ResourceData, meta interfa
 		}
 	}
 
+	if firewallInstance.FirewallImageId != "" && !intInSlice(cloudType, []int{goaviatrix.AWS, goaviatrix.AZURE}) {
```

Comment:
Should this be `goaviatrix.IsCloudType(cloudtype, goaviatrix.AWSRelatedCloudTypes|goavtiatrix.AzureRelatedCloudTypes` ?

Reply:
this will be updated after Matthew's PR is merged